src/custom/theme/baseTheme.tsx

```diff
@@ -25,6 +25,10 @@ export function colors(darkMode: boolean): Colors {
     bg1: darkMode ? '#1E1F2C' : '#FFFFFF',
     bg2: darkMode ? '#2C2D3F' : '#F7F8FA',
     bg3: darkMode ? '#1E1F2C' : '#EDEEF2',
+    bg4: darkMode ? '#021E34' : '#ffffff',
```

Comment:
bg4 - 7 come from develop (the branch i originally made this PR from before switching to release/1.3.0)